src/App.js

```diff
@@ -1,14 +1,243 @@
+                            <div className="nameInputContainer">
+                                <div className="titleText">
+                                    <h2>Who's eating tonight?</h2>
+                                </div>
+                                <div className="singleNameInput">
+                                    <label className='sr-only' htmlFor="nameOne">Name</label>
+                                    <input
+                                        type="text"
+                                        id='nameOne'
+                                        name='nameOne'
+                                        value={diners.nameOne}
+                                        placeholder='Name'
+                                        onChange={handleInputChange}
+                                        required
+                                    />
+
+                                </div>
+                                <div className="singleNameInput">
+                                    <label className='sr-only' htmlFor="nameTwo">Name</label>
+                                    <input
+                                        type="text"
+                                        id='nameTwo'
+                                        name='nameTwo'
+                                        value={diners.nameTwo}
+                                        placeholder='Name'
+                                        onChange={handleInputChange}
+                                        required
+                                    />
+                                </div>
+                                <div className="singleNameInput">
+                                    <label className='sr-only' htmlFor="nameThree">Name</label>
+                                    <input
+                                        type="text"
+                                        id='nameThree'
+                                        name='nameThree'
+                                        value={diners.nameThree}
+                                        placeholder='Name'
+                                        onChange={handleInputChange}
+                                        required
+                                    />
+
+                                </div>
+                                <div className="singleNameInput">
+                                    <label className='sr-only' htmlFor="nameFour">Name</label>
+                                    <input
+                                        type="text"
+                                        id='nameFour'
+                                        name='nameFour'
+                                        value={diners.nameFour}
+                                        placeholder='Name'
+                                        onChange={handleInputChange}
+                                        required
+                                    />
```

Comment:
It looks like the `singleNameInput` divs here are a great spot to use another component, passing the different diners down as props. That might also make it easy to vary the number of diners.